setup.py

```diff
@@ -712,65 +712,7 @@ def pxd(name):
 setup(name=DISTNAME,
       maintainer=AUTHOR,
       version=versioneer.get_version(),
-      packages=['pandas',
-                'pandas.api',
-                'pandas.api.types',
-                'pandas.compat',
-                'pandas.compat.numpy',
-                'pandas.core',
-                'pandas.core.dtypes',
-                'pandas.core.indexes',
-                'pandas.core.computation',
-                'pandas.core.reshape',
-                'pandas.core.sparse',
-                'pandas.core.tools',
-                'pandas.core.util',
-                'pandas.computation',
-                'pandas.errors',
-                'pandas.formats',
-                'pandas.io',
-                'pandas.io.json',
-                'pandas.io.sas',
-                'pandas.io.msgpack',
-                'pandas.io.formats',
-                'pandas.io.clipboard',
-                'pandas._libs',
-                'pandas._libs.tslibs',
-                'pandas.plotting',
-                'pandas.stats',
-                'pandas.types',
-                'pandas.util',
-                'pandas.tests',
-                'pandas.tests.api',
-                'pandas.tests.dtypes',
-                'pandas.tests.computation',
-                'pandas.tests.sparse',
-                'pandas.tests.frame',
-                'pandas.tests.generic',
-                'pandas.tests.indexing',
-                'pandas.tests.indexes',
-                'pandas.tests.indexes.datetimes',
-                'pandas.tests.indexes.timedeltas',
-                'pandas.tests.indexes.period',
-                'pandas.tests.internals',
-                'pandas.tests.io',
-                'pandas.tests.io.json',
-                'pandas.tests.io.parser',
-                'pandas.tests.io.sas',
-                'pandas.tests.io.msgpack',
-                'pandas.tests.io.formats',
-                'pandas.tests.groupby',
-                'pandas.tests.reshape',
-                'pandas.tests.series',
-                'pandas.tests.scalar',
-                'pandas.tests.tseries',
-                'pandas.tests.tseries.offsets',
-                'pandas.tests.plotting',
-                'pandas.tests.tools',
-                'pandas.tests.util',
-                'pandas.tools',
-                'pandas.tseries',
-                ],
+      packages=find_packages(include=['pandas', 'pandas.*']),
       package_data={'pandas.tests': ['data/*.csv'],
```

Comment:
I think there is also a utility to find various package data as well?

Reply:
Below is what I could find in the setuptools documentation on options to include package data. I went with the second suggestion and just included a wildcard entry for all of the extensions currently in use.

http://setuptools.readthedocs.io/en/latest/setuptools.html#including-data-files